New ships & outfits for the Unfettered (#7019)
endless-sky/endless-sky: data/hai/hai outfits.txt

outfit "Tripulse Shredder"
	category "Guns"
	cost 520000
	thumbnail "outfit/tripulse shredder"
	"mass" 42
	"outfit space" -42
	"weapon capacity" -42
	"gun ports" -1
	weapon
		sprite "projectile/pulse"
		sound "pulse"
		"hit effect" "pulse impact"
		"inaccuracy" 5
		"velocity" 15
		"lifetime" 30
		"reload" 6
		"burst count" 36
		"burst reload" 4
		"cluster"
		"firing energy" 25
		"firing heat" 60
		"shield damage" 45
		"hull damage" 34
	description "An early stage in the development of better and faster weaponry to rival that of the Wanderers, the Tripulse Shredder is essentially just three Pulse Cannons in one single, larger housing. Integrating advances in cooling capabilities has reduced the heat production per shot significantly at the cost of drawing just a touch more power. The quicker firing speed, however, still generates much more total heat in the same time. Additionally, the single shared housing results in a less stable firing pattern, limiting it to bursts with short cooldown gaps in between."

> 	description "	Initially designed with the Sea Scorpion in mind, the Tripulse Shredder's unique properties compared to the standard Pulse Cannon make it ideal for hit and run tactics, and the Unfettered pilots most skilled with this weapon are able to time its needed rest time to sync with the end of their initial attack, allowing them to harass freighters or even cripple their smaller escort ships with its high damage before either fleeing or coming about for a second pass. Furthermore, when this technique is utilized properly, the weapon allows for small, agile ships to damage and even defeat heavier ships by making use of their high mobility to compensate for its low range."

Author: These weapons are going to be in the campaign, and will be not be unlocked at any time, you can only steal them (and are encouraged to do so)
I like long descriptions because nobody has to read them but they add depth to the game

	"velocity scale" -.05

outfit "Ionic Blaster Prototype"
	category "Guns"
	cost 530000
	thumbnail "outfit/hai ionic blaster"
	"mass" 28
	"outfit space" -28
	"weapon capacity" -28
	"gun ports" -1
	weapon
		sprite "projectile/ion blast"
			"frame rate" 5
		sound "ion blast"
		"hit effect" "ion impact" 2
		"inaccuracy" 4
		"velocity" 18
		"lifetime" 40
		"reload" 50
		"firing energy" 170
		"firing force" 20
		"firing heat" 220
		"hit force" 120
		"shield damage" 150
		"hull damage" 45
		"ion damage" 1.5
	description "Since the start of the war with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that goal, they began developing the Ionic Blaster, which does a small amount of ion damage that scrambles the weapons of the target, sometimes disabling it if its power generation is low enough. It also has enough raw damage to be useful in all situations."

> 	description "The main focus they had in mind when developping the weapon was making sure they could fill shield beetles with them, unlike the bigger ion cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."

Reviewer: ```suggestion
	description "The main focus they had in mind when developping the weapon was making sure they could fill Shield Beetles with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."
```

> 	description "The main focus they had in mind when developping the weapon was making sure they could fill Shield Beetles with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."

Reviewer: I'm not sure if this is the intended meaning, but I edited it so that the description would be clearer. Apologies if I changed the meaning.
```suggestion
	description "The main focus the Unfettered had in mind when developing the weapon was making sure they could fill all of a Shield Beetle's gun ports with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."
```

Author: Well they can actually fill of a SB's gun ports with them, and it isnt far from the truth. Also you're right about it being clearer.

> 	description "The main focus the Unfettered had in mind when developing the weapon was making sure they could fill all of a Shield Beetle's gun ports with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."

Reviewer: ```suggestion
	description "The main focus the Unfettered had in mind when developing the weapon was making sure they could fill all of a Shield Beetle's gun ports with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."
```
```suggestion
	description "The main focus the Unfettered had in mind when developing the weapon was making sure they could fill all of a Shield Beetle's gun ports with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass-produce them, so it is only a prized possession of a lucky few."
```

outfit "Ionic Turret Prototype"
	category "Turrets"
	cost 1260000
	thumbnail "outfit/hai ionic turret"
	"mass" 58
	"outfit space" -58
	"weapon capacity" -58
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/ion blast"
			"frame rate" 5
		"hardpoint sprite" "hardpoint/hai ionic turret"
		"hardpoint offset" 16.
		sound "ion blast"
		"hit effect" "ion impact" 2
		"inaccuracy" 2
		"turret turn" 2.5
		"velocity" 12
		"lifetime" 50
		"reload" 30
		"firing energy" 200
		"firing force" 10
		"firing heat" 180
		"hit force" 60
		"shield damage" 80
		"hull damage" 45
		"ion damage" 4.5

> 	description "This is a dual turret version of the Ionic Blaster Prototype, the first weapon that the Unfetttered developed independently. However, they did not stop there, and refined the ion in the shots to be more coherent and thus apply more disruption, at the cost of damage. This weapon quickly disables any small ships that lack power generation, and jamms the weapons of others. It is surprisingly small for a turret, because it requires less cooling to function properly."

Reviewer: ```suggestion
	description "This is a dual turret version of the Ionic Blaster Prototype, the first weapon that the Unfetttered developed independently. However, they did not stop there, and refined the ion in the shots to be more coherent, and thus apply more disruption, at the cost of damage. This weapon quickly disables any small ships that lack power generation, and jams the weapons of others. It is surprisingly small for a turret, because it requires less cooling to function properly."
```

> 	description "This is a dual turret version of the Ionic Blaster Prototype, the first weapon that the Unfetttered developed independently. However, they did not stop there, and refined the ion in the shots to be more coherent, and thus apply more disruption, at the cost of damage. This weapon quickly disables any small ships that lack power generation, and jams the weapons of others. It is surprisingly small for a turret, because it requires less cooling to function properly."

Reviewer: ```suggestion
	description "This is a dual turret version of the Ionic Blaster Prototype, the first weapon that the Unfetttered developed independently. However, they did not stop there - they refined the ion in the shots to be more coherent, and thus apply more disruption, at the cost of dealing less damage. This weapon quickly disables any small ships that lack power generation, and jams the weapons of others. It is surprisingly small for a turret, because it requires less cooling to function properly."
```

	"velocity scale" -.05

outfit "Ionic Blaster Prototype"
	category "Guns"
	cost 530000
	thumbnail "outfit/hai ionic blaster"
	"mass" 19
	"outfit space" -19
	"weapon capacity" -19
	"gun ports" -1
	weapon
		sprite "projectile/ion bolt"
			"frame rate" 5
		sound "ion blast"
		"hit effect" "ion impact"
		"inaccuracy" 4
		"velocity" 18
		"lifetime" 30
		"reload" 30
		"firing energy" 72
		"firing force" 12
		"firing heat" 108
		"hit force" 72
		"shield damage" 70
		"hull damage" 30
		"ion damage" 1

> 	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon, but still able to apply the same weapon jamming and energy draining ion effects, albeit to a much lesser extent. It also has enough raw damage to be useful in all situations."

Reviewer: ```suggestion
	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. It is much smaller than the venerable Ion Cannon, but still able to apply the same weapon jamming and energy draining ion effects, albeit to a much lesser extent. It also has enough raw damage to be useful in all situations."
```

Reviewer 2: I think it'd be better to replace the full stop at the end of this sentence to a comma and combine it with the next one rather than having both begin with "it."

Author: Is this what you meant?

Reviewer 3: ```suggestion
	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects, albeit to a much lesser extent, it also has enough raw damage to be useful in all situations."
```

@Hurleveur , I thought this was what @warp-core meant?

Breaks down to "much smaller... it also has enough..."

This way also needs a comma removed and the second instance of ion becomes more redundant in the pacing change.

Author: I applied that change

	description "The design goal for the Ionic Blaster was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles. The result is also small enough to be equipped on the Unfettered's new fighters."
	description "This design cannot yet be mass produced, but those that have been produced are prized by the few lucky enough to possess them."

outfit "Ionic Turret Prototype"
	category "Turrets"
	cost 1260000
	thumbnail "outfit/hai ionic turret"
	"mass" 58
	"outfit space" -58
	"weapon capacity" -58
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/ion blast"
			"frame rate" 5
		"hardpoint sprite" "hardpoint/hai ionic turret"
		"hardpoint offset" 16.
		sound "ion blast"
		"hit effect" "ion impact" 5
		"inaccuracy" 2
		"turret turn" 2.4
		"velocity" 12
		"lifetime" 70
		"reload" 70
		"firing energy" 371
		"firing force" 8
		"firing heat" 259
		"hit force" 60
		"shield damage" 100
		"hull damage" 50
		"ion damage" 14

> 	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret mounted, enabling it to fire at enemies in any direction, not just those directly ahead."

Reviewer: ```suggestion
	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret-mounted, enabling it to fire at enemies in any direction and not just those directly ahead."
```

Reviewer 2: I have no opinion on the hyphenation, but I don't think the comma should be replaced with 'and'.

Author: I applied the hyphenation

	"velocity scale" -.05

outfit "Ionic Blaster Prototype"
	category "Guns"
	cost 530000
	thumbnail "outfit/hai ionic blaster"
	"mass" 19
	"outfit space" -19
	"weapon capacity" -19
	"gun ports" -1
	weapon
		sprite "projectile/ion blast"
			"frame rate" 5
		sound "ion blast"
		"hit effect" "ion impact"
		"inaccuracy" 4
		"velocity" 18
		"lifetime" 30
		"reload" 30
		"firing energy" 72
		"firing force" 12
		"firing heat" 108
		"hit force" 72
		"shield damage" 70
		"hull damage" 30
		"ion damage" 1
	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources, to more effectively make their point. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects to a useful degree. Some of that output has been traded to do enough raw damage to be useful in all situations."

> 	description "	This aligns with the design goal for the Ionic Blaster, which was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles with maximal efficiency, and still be useful on other smaller vessels. This goal has resulted in a design which cannot yet be mass produced, but is also small enough to be equipped on the Unfettered's new fighters. These weapons have consequently become the prized possessions of the few lucky enough to possess them."

Author: The other ones I'm fone with removing fine but this one is a prototype and is never explained anywhere else
It'll never actually be produced

outfit "Ionic Turret Prototype"
	category "Turrets"
	cost 1260000

> 	thumbnail "outfit/hai ionic turret"

Author: Because becca did not have the time to change that yet - these are not accessible in game though so it should be fine.
This PR was mostly done to figure out the stats of the weapons of the Unfettered campaign

	"velocity scale" -.05

outfit "Ionic Blaster Prototype"
	category "Guns"
	cost 530000
	thumbnail "outfit/hai ionic blaster"
	"mass" 19
	"outfit space" -19
	"weapon capacity" -19
	"gun ports" -1
	weapon
		sprite "projectile/ion blast"
			"frame rate" 5
		sound "ion blast"
		"hit effect" "ion impact"
		"inaccuracy" 4
		"velocity" 18
		"lifetime" 30
		"reload" 30
		"firing energy" 72
		"firing force" 12
		"firing heat" 108
		"hit force" 72
		"shield damage" 70
		"hull damage" 30
		"ion damage" 1

> 	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects, albeit to a much lesser extent, it also has enough raw damage to be useful in all situations."
> 	description "	The design goal for the Ionic Blaster was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles. The result is also small enough to be equipped on the Unfettered's new fighters."
> 	description "	This design cannot yet be mass produced, but those that have been produced are prized by the few lucky enough to possess them."

Reviewer 4: ```suggestion
	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources, to more effectively make their point. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects to a useful degree. Some of that output has been traded to do enough raw damage to be useful in all situations."
	description "	This aligns with the design goal for the Ionic Blaster, which was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles with maximal efficiency, and still be useful on other vessels. This goal has resulted in a design which cannot yet be mass produced, but is also small enough to be equipped on the Unfettered's new fighters. These weapons have consequently become the prized possessions of the few lucky enough to possess them."
```

Suggested reworking of that.

Also, are these used before the Solifuge is developed, or exclusively after? If before then the line about fighters needs to be tweaked.

Author: Before but only in very limited quantities, and they were designed for those fighters so maybe I just make it more ambiguous

outfit "Ionic Turret Prototype"
	category "Turrets"
	cost 1260000
	thumbnail "outfit/hai ionic turret"
	"mass" 58
	"outfit space" -58
	"weapon capacity" -58
	"turret mounts" -1
	"required crew" 1
	weapon
		sprite "projectile/ion bolt"
			"frame rate" 5
		"hardpoint sprite" "hardpoint/hai ionic turret"
		"hardpoint offset" 16.
		sound "ion blast"
		"hit effect" "ion impact" 5
		"inaccuracy" 2
		"turret turn" 2.4
		"velocity" 12
		"lifetime" 70
		"reload" 70
		"firing energy" 371
		"firing force" 8
		"firing heat" 259
		"hit force" 60
		"shield damage" 100
		"hull damage" 50
		"ion damage" 14
	description "This weapon improves upon the ionization capabilities of the Ion Cannon, while also being turret-mounted, enabling it to fire at enemies in any direction, not just those directly ahead."

> 	description "	This design cannot yet be mass produced, but those that have been produced are prized by the few lucky enough to possess them."

Reviewer 4: I'm assuming the intent is that a shield beetle can fill its weapon space nearly perfectly with Ionic blasters, and some combination of ionic turrets + AM. Can you confirm?

Also is display name a thing yet?
If so these should just be "Ionic <thing>" and have the "Prototype" added as a display name so that you can just remove that later when/if they go into mass production.

Author: It isnt a thing yet and I'll be making new outfits after these that will be the non prototype version; they wont have the same stats and the player will have a say for said stats

Author: Yeah a shield beetle can fit 8 of these + 2 turrets + either 2 AM or another turret

	"quantum keystone" 1
	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

outfit "Value Detector"
	category "Systems"
	cost 260000
	thumbnail "outfit/value detector"
	"mass" 3
	"outfit space" -3
	"cargo scan power" 33
	"cargo scan speed" 4
	"tactical scan power" 15

> 	description "In order to know which ship is worth plundering, or presents less risks to do so, the Unfettered use this scanner. It is quite rudimentary compared to normal Hai technology, but it still does what it's supposed to do: get quick information on what ships are carrying from afar. The Unfettered also prefer to know more detailed information about their enemy, as they often try new technologies on their victims in order to test their effectiveness."

Reviewer 2: ```suggestion
	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons.. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar. It also provides more detailed information about the tactical condition of enemy vessels; useful when testing new weapons."
```

	"quantum keystone" 1
	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

outfit "Value Detector"
	category "Systems"
	cost 260000
	thumbnail "outfit/value detector"
	"mass" 3
	"outfit space" -3
	"cargo scan power" 22
	"cargo scan speed" 4
	"tactical scan power" 27

> 	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons.. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar. It also provides more detailed information about the tactical condition of enemy vessels; useful when testing new weapons."

Reviewer 3: ```suggestion
	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar. It also provides more detailed information about the tactical condition of enemy vessels; useful when testing new weapons."
```

	"quantum keystone" 1
	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

outfit "Value Detector"
	category "Systems"
	cost 260000
	thumbnail "outfit/value detector"
	"mass" 3
	"outfit space" -3
	"cargo scan power" 22
	"cargo scan speed" 4
	"tactical scan power" 27

> 	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar and provide more detailed information about the tactical condition of enemy vessels. This dual function is useful when testing new weapons."

Author: Well you test the weapons on the right enemy, of course!

> 	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar and provide more detailed information about the tactical condition of enemy vessels. This dual function is useful when testing new weapons."

Reviewer 5: Then that's not weapon testing, that's fielding experimental weaponry.

> 	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar and provide more detailed information about the tactical condition of enemy vessels. This dual function is useful when testing new weapons."

Author: Renamed it to that!

	"quantum keystone" 1
	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

outfit "Value Detector"
	category "Systems"
	cost 260000
	thumbnail "outfit/value detector"
	"mass" 3
	"outfit space" -3
	"cargo scan power" 22
	"cargo scan speed" 4

> 	"tactical scan power" 27

Reviewer 4: Is this designed to make them scan at exactly the same speed?
That would be sensible I think here.

Author: The power only is for the range of the scan, and you first want to know how vulnerable the ship is before learning about its cargo
You cant actually tactical scan a ship it just reveals the stats

> 	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar. It also provides more detailed information about the tactical condition of enemy vessels; useful when testing new weapons."

Reviewer 4: ```suggestion
	description "The Unfettered developed this scanner in order to determine which ships are worth plundering, or which lack the capacity to stand up to their ion weapons. Some Hai consider it rather rudimentary, but it is sufficient to identify the cargo on a ship from afar and provide more detailed information about the tactical condition of enemy vessels. This dual function is useful when testing new weapons."
```
